src/task3.py

```diff
@@ -52,5 +52,7 @@ def rankings():
     print("PROGRAM \t  \t | \t RANK \t | \t TIME ELAPSED")
     i = 1
     for k, v in sorted_ranks.items():
-        print(k + " \t   " + str(i) + " \t   \t " + str(v))
```

Comment:
fixed the formatting of the ranking function

- Before:
<img width="459" alt="image" src="https://user-images.githubusercontent.com/49820108/191928693-328f549b-e25e-49bf-a9b7-f2bb9b1fa774.png">

- After:

<img width="444" alt="image" src="https://user-images.githubusercontent.com/49820108/191928951-561fce0e-2ca6-47ea-ab28-000c63bc80b4.png">